### ISSUE TITLE
PWR Wallet issue

### ISSUE THREAD

**Issue by sourmack1** [PWRcoin/PWRcoinClassic#33]
Hi,

My PWR Wallet went out of Sync many months ago and it synced again. I have tried opening it today and get "Assertion failed error" and below is the screenshot for your reference.

![image](https://user-images.githubusercontent.com/55728461/65480556-54959400-ded5-11e9-8770-9eef9f1e1891.png)

This is the wallet version i am using: pwrcoin-qt-x32-v3.0.0

Can you please advise how to resolve this issue and sync my wallet afterwards?

I have an old backup of pwrencrypted.dat file so should be able to restore everything from scratch if needed but not sure on the process to follow.

Regards,
Sourmack

**Comment by ghobson2013** [comment]
Hi,

1-  verify you have enough space left on C: drive to completely store the blockchain database , at least 2Gb free space.
2 - stop your wallet
3 - backup wallet.dat in %APPDATA%/Roaming/pwrcoin3/ 
4 - delete all files in the %APPDATA%/Roaming/pwrcoin3/ except wallet.dat and pwrcoin.conf if there is one.
5 - download https://github.com/PWRcoin/PWRcoin/releases/download/v3.1.0/pwr-boot-strap.zip
unzip its content inside %APPDATA%/Roaming/pwrcoin3/
6 - start wallet back up and let it sync up.

**Comment by sourmack1** [comment]
Hi,
Thank you for the quick response. 
I have followed the steps and the wallet is now synching however the wallet now shows zero coins. Is this expected and will it resolve once the sync is complete?



    On Tuesday, 24 September 2019, 02:40:37 pm AEST, Gary Hobson <notifications@github.com> wrote:  
 
 
Hi,

1- verify you have enough space left on C: drive to completely store the blockchain database , at least 2Gb free space.
2 - stop your wallet
3 - backup wallet.dat in %APPDATA%/Roaming/pwrcoin3/
4 - delete all files in the %APPDATA%/Roaming/pwrcoin3/ except wallet.dat and pwrcoin.conf if there is one.
5 - download https://github.com/PWRcoin/PWRcoin/releases/download/v3.1.0/pwr-boot-strap.zip
unzip its content inside %APPDATA%/Roaming/pwrcoin3/
6 - start wallet back up and let it sync up.

**Comment by ghobson2013** [comment]
yes this is normal , the wallet is downloading the blocks, and will increase your balance as it finds your transactions... you can always check on the block explorer to see if your funds are still there.
